src/components/PDF/ApplicationForm.tsx

```diff
@@ -48,21 +48,22 @@ class ApplicationForm extends Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      formQuestions: undefined,
+      fields: undefined,
       formAnswers: {},
       pdfApplication: undefined,
-      buttonState: '',
+      title: '',
+      description: '',
       submitSuccessful: false,
       currentPage: 0,
       numPages: 0,
-      startDate: new Date(),
       formError: false,
     };
     this.handleChangeFormValueTextField = this.handleChangeFormValueTextField.bind(this);
     this.handleChangeFormValueRadioButton = this.handleChangeFormValueRadioButton.bind(this);
     this.handleChangeFormValueCheckBox = this.handleChangeFormValueCheckBox.bind(this);
     this.handleChangeFormValueListBox = this.handleChangeFormValueListBox.bind(this);
-    this.onSubmitFormQuestions = this.onSubmitFormQuestions.bind(this);
+    this.handleChangeFormValueDateField = this.handleChangeFormValueDateField.bind(this);
```

Comment:
Don't need to bind if using arrow functions

Reply:
Done.